package.json

```diff
@@ -14,16 +14,20 @@
   "dependencies": {
     "@auth0/auth0-react": "2.1.1",
     "@aws-amplify/ui-react": "^5.0.4",
+    "@babel/core": "7.0.1",
+    "@babel/plugin-syntax-flow": "^7.14.5",
+    "@babel/plugin-transform-react-jsx": "^7.14.9",
     "@graphql-tools/graphql-file-loader": "7.3.10",
     "@graphql-tools/load": "7.5.9",
     "@material-ui/core": "4.12.4",
     "@material-ui/icons": "4.11.3",
     "@material-ui/lab": "4.0.0-alpha.61",
+    "@matheusluizn/react-google-login": "^5.1.6",
```

Comment:
Based on the issues in `react-google-login` is seems like most people are transitioning to the [@react-oath/google](https://www.npmjs.com/package/@react-oauth/google) package.

Reply:
@mschile trying to switch to this one now - there are some differences in implementation between the two packages. Any objection with leaving this as-is in this PR since I don't have a ton of time to dedicate to this?